contacktbook/DataBase.cs

```diff
@@ -45,17 +52,43 @@ public void updateContact(Contact contact)
                                                               contact.Tel1, contact.Tel2, contact.Email);
                 SQLiteCommand updateCommand = new SQLiteCommand(updateContactTableQuery, connection);
                 updateCommand.ExecuteNonQuery();
+                }
+
+                catch (SQLiteException e)
+                {
+                    LoggerClass logger = new LoggerClass();
+                    logger.writeLog(" SQL Statement Error");
+                    MessageBox.Show("Error!");
+
+                }
+
+                catch (Exception e)
+                {
+                    LoggerClass logger = new LoggerClass();
+                    logger.writeLog(" Unknown error");
+                    MessageBox.Show("Error!");
+                }
             }
         }
-
+ 
         public void deleteContact(Contact contact)
         {
             using (var connection = new SQLiteConnection(connString))
             {
-                connection.Open();
-                string deleteContactTableQuery = String.Format("DELETE FROM contacts WHERE first_name = '{0}' AND second_name = '{1}'", contact.FirstName, contact.SecondName);
-                SQLiteCommand deleteCommand = new SQLiteCommand(deleteContactTableQuery, connection);
-                deleteCommand.ExecuteNonQuery();
+                try
+                {
+                    connection.Open();
+                    string deleteContactTableQuery = String.Format("DELETE FROM contacts WHERE first_name = '{0}' AND second_name = '{1}'", contact.FirstName, contact.SecondName);
+                    SQLiteCommand deleteCommand = new SQLiteCommand(deleteContactTableQuery, connection);
+                    deleteCommand.ExecuteNonQuery();
+
+                }
+                catch (Exception e)
+                {
+                    LoggerClass logger = new LoggerClass();
+                    logger.writeLog(" Delete querry error");
+                    MessageBox.Show("Error!");

```

Comment:
This is a good idea to show than something bad happen to user. But it is worth to do it in not so scary manner. 
For instance, message can be "DB restore is temporary unavailable. Please try again later"